opennms-base-assembly/src/main/filtered/etc/create.sql

```diff
@@ -343,7 +343,7 @@ CREATE TABLE scanreports (
     CONSTRAINT scanreports_monitoringlocations_fkey FOREIGN KEY (location) REFERENCES monitoringlocations (id) ON DELETE CASCADE ON UPDATE CASCADE
 );
 
-CREATE UNIQUE INDEX scanreports_id_idx on scanreport(id);
+CREATE UNIQUE INDEX scanreports_id_idx on scanreports(id);
```

Comment:
Not sure if...

Reply:
Oh, come on, it's not really hurting anyone.